matcher/service/celery_tasks.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from matcher import Matcher
+from matcher.core import MatcherStatus
+from matcher.service.openreview_interface import (
+    ConfigNoteInterface,
+    Deployment,
+)
+from matcher.service.server import celery_app as celery
+
+
+@celery.task(
+    name="matching", track_started=True, bind=True, time_limit=3600 * 24
```

Comment:
what is time limit for? can we store this value in the config file?

Reply:
The task has to finish before this time limit otherwise celery will stop the task and mark it as failure.

Yes, we can store this value in the config.